server/routes/service-provider/referrals/supplier-assessment/attendance/postSessionAttendanceFeedbackPresenter.ts

```diff
@@ -0,0 +1,30 @@
+import Appointment from '../../../../../models/appointment'
+import DeliusServiceUser from '../../../../../models/delius/deliusServiceUser'
+import { FormValidationError } from '../../../../../utils/formValidationError'
+import PostSessionAttendanceFeedbackPresenterComponent from '../../../appointment/post-session-feedback/attendance/postSessionAttendanceFeedbackPresenterComponent'
+
+export default class PostInitialAppointmentAttendanceFeedbackPresenter {
+  constructor(
+    private readonly appointment: Appointment,
+    private readonly serviceUser: DeliusServiceUser,
+    private readonly error: FormValidationError | null = null,
+    private readonly userInputData: Record<string, unknown> | null = null
+  ) {}
+
+  private readonly text = {
+    title: `Add feedback`,
+    subTitle: 'Appointment details',
+    attendanceQuestion: `Did ${this.serviceUser.firstName} attend the initial assessment appointment?`,
```

Comment:
are SA appointments alway in person?

Reply:
Well I asked this question to Sheida. There is still disagreements on how to present this to the user. I said to her that I will keep it as this way until the agreement has been made.